tools/shared.py

```diff
@@ -39,6 +39,7 @@
 EXPECTED_NODE_VERSION = (4, 1, 1)
 EXPECTED_BINARYEN_VERSION = 94
 SIMD_FEATURE_TOWER = ['-msse', '-msse2', '-msse3', '-mssse3', '-msse4.1', '-msse4.2', '-mavx']
```

Comment:
Maybe SIMD_FEATURE_TOWER should be renamed SIMD_SSE_FEATURE_TOWER or something to make it clear that it does not cover all SIMD options.